app/script/view_model/content/HistoryExportViewModel.js

```diff
@@ -81,7 +87,7 @@ z.viewModel.content.HistoryExportViewModel = class HistoryExportViewModel {
         .then(archive => archive.generateAsync({type: 'blob'}))
         .then(archiveBlob => {
           const timestamp = new Date().toISOString().substring(0, 10);
-          const filename = `Wire-${userName}-Backup_${timestamp}.desktop_wbu`;
+          const filename = `Wire-${userName}-Backup_${timestamp}.${this.CONFIG.EXPORT_FILE_EXTENSION}`;
```

Comment:
If it is static, then it should be `z.viewModel.content.HistoryExportViewModel.CONFIG.EXPORT_FILE_EXTENSION`